hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -146,10 +147,12 @@ enum FreonOps {
   @Option(
       names = {"--key-size", "--keySize"},
       description = "Specifies the size of Key in bytes to be created. Full" +
-          " name --keySize will be removed in later versions.",
-      defaultValue = "10240"
+          " name --keySize will be removed in later versions. You can" +
+          " specify the size using data units like 'GB', 'MB', 'KB', etc.",
+      defaultValue = "10GB",
```

Comment:
Should be `10KB`?

Reply:
Corrected this typo to `10KB`, thank you!